templates/web/base/admin/problem_row.html

```diff
@@ -38,6 +38,22 @@
             [%- IF problem.is_closed %]<br>[% prettify_state('closed') %]: [% PROCESS format_time time=problem.lastupdate %][% END -%]
             [%- IF problem.is_open %]<br>[% loc('Last&nbsp;update:') %] [% PROCESS format_time time=problem.lastupdate %][% END -%]
         </small></td>
+        [%- IF c.user.has_permission_to('planned_reports', c.cobrand.body.id ) %] 
+        [%# see report's assigned user %]
+        <td>
+            <span>
+            [%- IF problem.user_planned_reports -%] [%# this report is on a shortlist %]
+                [%- IF problem.shortlisted_user.name -%]
```

Comment:
Similar to before, could be `[% problem.shortlisted_user.name OR problem.shortlisted_user.username %]`